src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -44,10 +44,12 @@ class TestConcurrencyManager : public TestLoadManagerBase,
             params.async, params.streaming, params.batch_size,
             params.max_threads, params.max_concurrency, params.sequence_length,
             params.shared_memory_type, params.output_shm_size,
-            params.start_sequence_id, params.sequence_id_range,
-            params.string_length, params.string_data, params.zero_input,
-            params.user_data, GetParser(), GetFactory())
+            params.start_sequence_id, params.sequence_id_range, GetParser(),
+            GetFactory())
   {
+    InitManager(
```

Comment:
So in the production code, `InitManager()` is called after the constructor, but in the testing code, it's called in the (albeit subclass's) constructor?

Reply:
yes. I need to be able to overload it, which isnt possible when called from the constructor.
The new constructor I made, does not call it specifically to avoid this issue but I left the other version of the test constructor alone to not break previous testing.